packages/esm-tb-app/translations/fr.json

```diff
@@ -0,0 +1,8 @@
+{
+  "activeDrCases": "Active DR Cases",
```

Comment:
Lets translate these, for example the first value should be `Cas de reprise après sinistre actifs`